R/multilevelannotationstep4.R

```diff
@@ -232,7 +232,7 @@ multilevelannotationstep4 <- function(outloc,
 
             # chemids<-c("HMDB00277","HMDB00222","HMDB00043")
 
-            # stopCluster(cl)
+            stopCluster(cl)

```

Comment:
I noticed the cluster is actually not used in the step 4 at all, so I guess there is no need to stop it. Actually, I'm planning to completely remove it. Or is there any particular reason for this change?

Reply:
It is just to stop it since it is actually started/created a couple lines further above.

We will remove that at some point but for now it is good, since otherwise there are leftover rpocesses.